components/rich_text_editor/extensions/emoji/EmojiSuggestion.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div
+    class="dt-emoji-suggestion"
+  >
+    <dt-emoji
+      size="200"
+      :code="item"
+    />
+    <div class="dt-emoji-suggestion--text">
+      {{ item }}
+    </div>
+  </div>
```

Comment:
can probably use dt-stack with gap here instead of margin. Always avoid using margin for spacing between items if possible.